packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -19,8 +19,24 @@ export interface PaletteRange {
   900: string;
 }
 
+export interface PaletteText {
+  heading: React.CSSProperties['color'];
```

Comment:
```javascript
enum CssTypes {
  text: React.CSSProperties['color'];
  backgroundColor: React.CSSProperties['backgroundColor'];
  ...
}
```

What do you think about creating such enumeration and use it throughout interfaces?

Reply:
It should be `interface` because we want it to be augmentable from the application side.